src/backend/utils/adt/gp_optimizer_functions.c

```diff
@@ -16,47 +16,78 @@
 #include "funcapi.h"
 #include "utils/builtins.h"
 
-extern Datum EnableXform(PG_FUNCTION_ARGS);
+typedef Datum (*gp_opt_version_func) (void);
+
+/*
+ * A stub function to call if no optimizer function is found.
+ */
+static Datum
+gp_optimizer_function_stub(PG_FUNCTION_ARGS)
+{
+	return CStringGetTextDatum("Server has been compiled without ORCA");
```

Comment:
compiled without ORCA or ORCA is not in shared lib?

Reply:
I've left the old message. But yes, I guess it could be changed to reflect actual situation better.